src/Agoda.Analyzers/RuleContent/AG0035PreventUseOfMachineName.html

```diff
@@ -0,0 +1,11 @@
+﻿<p>
+    Do not use <code>Environment.MachineName</code>.
```

Comment:
`Do not use <code>Environment.MachineName</code> or <code>Server.MachineName</code>.`